src/com/google/common/css/JobDescriptionBuilder.java

```diff
@@ -324,10 +327,20 @@ public JobDescriptionBuilder setAllowUnrecognizedFunctions(boolean allow) {
     return this;
   }
 
+  public JobDescriptionBuilder setAllowDuplicateDeclarations(boolean allow) {
```

Comment:
I think this should go just before `allowDuplicateDeclarations`. Somehow `allowUnrecognizedFunctions` managed to slip between the two.